docs/modules/tiles/api-reference/tileset-3d.md

```diff
@@ -169,13 +169,13 @@ See the [properties schema reference](https://github.com/AnalyticalGraphicsInc/3
 
 ### maximumScreenSpaceError : Number
 
-The maximum screen space error used to drive level of detail refinement. This value helps determine when a tile refines to its descendants, and therefore plays a major role in balancing performance with visual quality.
+Threshold that controls the level of detail of loaded tiles. A higher value means tile traversal stops early, displaying lower quality tiles (but much faster load times & less bandwidth used), because we're using a high "error tolerance". A lower value means lower tolerance for error, so traversal goes deeper in the tree and displays higher quality tiles. 
```

Comment:
In the spirit of an API audit, I am wondering if renaming this to some variation of `errorTolerance` would be helpful? That would make it clearer what higher and lower values mean.

Reply:
I agree `errorTolerance` makes a lot more sense. This one is a bit tricky because `maximumScreenSpaceError` is the technical term used by the 3D Tiles spec, so it's likely the same term that's present in other engines that support 3D Tiles (Unreal, Unity, etc). But that doesn't mean loadersgl can't abstract this away and expose a more user-friendly higher level version of it.

For example, `errorTolerance` would be some %, or just 3 settings like "low", "medium", "high", where "low" gives you 32, "medium" gives you 8, and "high" gives you 2. Could be called "levelOfDetail" too. I think those numbers roughly correspond to what most users would use in the real world, and the `maxSSE` could still be exposed for those who want that low level 3D Tileset control maybe.

Reply:
OK thanks, if it is in the spec we better keep it then.